src/apps/callNixWithD2N/default.nix

```diff
@@ -0,0 +1,21 @@
+{
+  # dream2nix deps
+  dream2nixWithExternals,
+  # nixpkgs deps
+  writers,
+  nix,
+  ...
+} @ args:
+writers.writeBash
+"callNixWithD2N"
+''
+  ${nix}/bin/nix ''${@:1:$#-1} --impure --expr "
+    let
+      nixpkgs = <nixpkgs>;
+      l = (import \"\''${nixpkgs}/lib\") // builtins;
+      dream2nix = import ${dream2nixWithExternals} {
+        config = ''${dream2nixConfig:-"{}"};
```

Comment:
The current `dream2nix` config should not be used here since in some cases (like in `translate` app) the config is changed in order to make sure the generated `resolve` scripts put the `dream-lock.json` in the correct path the user passed. Of course we could approach this in different ways (like making `resolve` scripts take a path) but I figured I would just keep everything as is and just set different values in config.

We could default to the current `configFile` instead of empty attrset though, that would be better.

Reply:
It now defaults to the current `configFile` instead of empty attrset.